transformer/src/main.py

```diff
@@ -82,15 +80,13 @@ def transform_row(resource_id, row, time_refresh_analysis=3600):
     logger.debug("Create FHIR Doc", extra={"resource_id": resource_id})
     fhir_document = transformer.create_fhir_document(data, analysis)
 
-    logger.debug(f"Fhir document: {fhir_document}", extra={"resource_id": resource_id})
-
     return fhir_document
 
 
 @app.route("/transform", methods=["POST"])
 def transform():
     body = request.get_json()
-    logger.info(f"Transform from API: {body}")
+    logger.info(f"Transform from API")
```

Comment:
it'd be nice to have at least the resource ID and maybe even the primary key value

Reply:
The primary key value is hard to get at this point, we have it only after having analyzed the mapping...